libs/plugins/scully-plugin-critical-css/src/lib/plugins-scully-plugin-critical-css.ts

```diff
@@ -55,6 +61,7 @@ const criticalCssPlugin = async (incomingHtml: string, route: HandledRoute) => {
         /** we will minify the inlined css */
         minify: true,
       },
+      ignore: settings.ignore,
```

Comment:
Is it safe to set ignore to undefined?
This will happen when a user doesn't provide the setting.

Reply:
I believe it is safe, I've copied the plugin in one of my repos and used it both with and without the ignore key. Some of the examples here (https://github.com/addyosmani/critical#generate-and-inline-critical-path-css) do not set the ignore key. I can try to add a test for this if you want, I'd need to look at a similar test to start though.

Reply:
Ok, then it's a good solution.